app/test_main.py

```diff
@@ -1,3 +1,57 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestAnimalToHumanAge:
+    def test_result_list_length_should_be_equal_to_2(self) -> None:
+        assert len(get_human_age(1, 1)) == 2
+
+    @pytest.mark.parametrize(
+        "cat_age,dog_age,expected_human_ages",
+        [
+            pytest.param(0, 0, [0, 0], id="should be zeros if ages less than 15"),
+            pytest.param(14, 14, [0, 0], id="should be zeros if ages less than 15"),
```

Comment:
These two test cases are essentially testing the same scenario: when both cat and dog ages are less than 15, the result should be [0, 0]. Consider removing one of them to avoid redundancy and adhere to the DRY principle.